src/Chartjs/Model/Chart.php

```diff
@@ -38,19 +38,25 @@ public function __construct(string $type)
         $this->type = $type;
     }
 
-    public function setData(array $data)
+    public function setData(array $data): self
```

Comment:
can you please add the `@return $this` annotation?
also can you please check if some other UX component could benefit from fluent setters?

Reply:
> can you please add the `@return $this` annotation?
You mean the doc block? is this still required?

> also can you please check if some other UX component could benefit from fluent setters?
From what I can see the UX-Cropperjs package could also benefit from fluent setters. should I include the changes here or open a separate PR?

Reply:
> You mean the doc block? is this still required?

yep, they are more precise than just `: self`

> should I include the changes here

yes please, in the same PR

Reply:
Done ;)